src/context/SearchQueryContext.tsx

```diff
@@ -82,9 +84,10 @@ export const SearchQueryProvider = ({ children }: { children: React.ReactNode})
     current: page + 1,
     totalResults: queryResult.data?.hits?.total["value"] as unknown as number ?? null
   }
-
+const isLoading =  queryResult.isLoading
+  useEffect(()=>{console.log("test")},[isLoading])
   return (
-    <SearchQueryContext.Provider value={{ searchQuery, queryResult, makeQuery, handlePageChange, pagingInfo }} >
+    <SearchQueryContext.Provider value={{ searchQuery, queryResult, makeQuery, handlePageChange, pagingInfo, isLoading }} >
```

Comment:
the `isLoading` property already exists on `queryResult` object. 

No need to explicitly pass the property since 
```js
queryResult.isLoading
``` 
has it